supertokens_python/recipe/emailverification/types.py

```diff
@@ -12,8 +12,20 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
 
 class User:
     def __init__(self, user_id: str, email: str):
         self.user_id = user_id
         self.email = email
+
+
+class STResponse(ABC):
```

Comment:
- Change name to `APIResponse`.
- This class should be in `supertokens_python/types.py`

Reply:
done.